source/content/guides/switch-drupal-recommended-upstream.md

```diff
@@ -1,20 +1,22 @@
 ---
-title: Switch from Drupal 9 to Drupal with Composer Upstream
+title: Switch from Drupal 9 to Drupal Composer Managed Upstream
 description: Switch to the new Pantheon upstream to take advantage of the new structure and future updates.
 type: guide
 permalink: docs/guides/:basename
 cms: "Drupal"
 categories: [develop]
 tags: [composer, site, workflow]
-contributors: [kporras07]
-reviewed: "2022-04-03"
+contributors: [kporras07, jspellman814]
+reviewed: "2022-04-22"
 ---
 
-In this guide, we will convert an existing site from the Drupal 9 (`drupal9`) upstream to the new Drupal with Composer (`drupal-recommended`) upstream.
+In this guide, we will convert an existing site from the Drupal 9 (`drupal9`) upstream to the new Drupal Composer Managed (`drupal-composer-managed`) upstream.
 
 ## Overview
 
-Drupal 9 sites created on the platform prior to November 30, 2021 use the [Drupal 9](https://github.com/pantheon-upstreams/drupal-project) upstream. Based on community needs, we have released a new upstream. [Drupal with Composer](https://github.com/pantheon-upstreams/drupal-recommended) is now the default Drupal 9 upstream on the platform and users are encouraged to switch to it to take advantage of the improved structure and updates.
+Drupal 9 sites created on the platform prior to November 30, 2021 use the [Drupal 9](https://github.com/pantheon-upstreams/drupal-project) upstream. Drupal 9 sites created on the platform prior to April 27, 2021 use the [Drupal Recommended](https://github.com/pantheon-upstreams/drupal-recommended) upstream.
```

Comment:
This date will need to be adjusted once https://github.com/pantheon-systems/hermes/pull/3036 is merged.